src/shared/src/components/organisms/AltinnReceipt.tsx

```diff
@@ -141,22 +141,27 @@ export function ReceiptComponent(props: IReceiptComponentProps) {
         >
           {props.body}
         </Typography>
-        {props.titleSubmitted && (
-          <Typography
-            variant='h3'
-            style={{
-              paddingTop: '4.1rem',
-              paddingBottom: '0.5rem',
-              fontWeight: 600,
-            }}
-          >
-            {props.titleSubmitted}
-          </Typography>
-        )}
-        <AltinnAttachmentComponent
-          attachments={props.pdf}
-          id='attachment-list-pdf'
-        />
+        {props.pdf && props.pdf.length > 0 && (
+          <React.Fragment>
```

Comment:
```suggestion
          <>
```

Reply:
Neat trick! Didn't know about this, but it certainly looks nicer. :+1: